dashboard/public/frequency/frequency.html

```diff
@@ -4,6 +4,8 @@
 
 <script type="text/javascript" src="/frequency/frequency.js"></script>
 
+<div id="bubble"></div>
```

Comment:
This element is where the markdown instructions will minimize to.  Ideally it also wraps the non-markdown instructions, so the student can click them and show the markdown instructions again.